models.py

```diff
@@ -41,6 +41,10 @@ class Member(ndb.Model):
 	
 	def get_missions(self):
 		return Mission.query(Mission.runners == self.id).order(Mission.start).fetch(limit=None)
+
+	def get_mission_ids(self):
+		missions = Mission.query(Mission.runners == self.id).order(Mission.start).fetch(limit=None)
+		return [mission.id for mission in missions]
```

Comment:
What is the benefit of this function existing?  I think it would make more sense to do the list comprehension in the one place you use `Member.mission_ids`.

Reply:
I am also concerned that the `get_mission_ids` function existing might suggest to someone new to the codebase that `get_mission_ids` is somehow less resource- or time-intensive than `get_missions`.